### PR TITLE
Closes #2

### LINKED ISSUE #2: Add a flask route to accept a image directly
Currently, the model only accepts an image link. Add a new route in flask that takes an image as a b64 object through an HTTP request. The response should remain the same.

### PR DESCRIPTION
**Pull request by anirudhRowjee**
Hi, this commit closes Issue #2.

Changelog:
1. Addition of method to accept base64 string as image and run the model
   on it
2. Addition of a new route to the app, to expose this new method

Closes #2

### CODE REVIEW

File: backend/app.py [hunk 1]
```diff
@@ -2,33 +2,54 @@
...
+    data = request.json
+    ans = search_by_url(data["url"])
+    print(ans)
+    return {"prediction": ans}
 
 
-@app.route('/api/url/bear', methods=['POST'])
+@app.route("/api/url/bear_base64", methods=["POST"])
 @cross_origin()
 def bear_hunt():
```
Comment (srujandeshpande) [inline backend/app.py]: `AssertionError: View function mapping is overwriting an existing endpoint function: bear_hunt`

Comment (srujandeshpande) [reply backend/app.py]: Rename the function

**Review by srujandeshpande — CHANGES_REQUESTED** [review]
Change the function name, it seems fine other than that

File: backend/app.py (hunk 1 above)
Comment (anirudhRowjee) [reply backend/app.py]: A very silly mistake :rofl:

**Review by srujandeshpande — APPROVED** [review]
The program runs!

**Comment by srujandeshpande** [comment]
I have not tested it with a b64 image but the code looks correct. Great work!

**Comment by srujandeshpande** [comment]
!bounty 200

**Comment by anirudhRowjee** [comment]
Thank you!

**Comment by srujandeshpande** [comment]
!bounty 120

**Pull request merged**